src/App.js

```diff
@@ -10,11 +10,13 @@ import Calendar from './components/Calendar';
 const USERS = [
   { label: 'Admin', value: 'admin', role: 'admin' },
   { label: 'Sasha', value: 'sasha', role: 'user' },
-  { label: 'Roma', value: 'roma', role: 'user' }
+  { label: 'Roma', value: 'roma', role: 'user' },
+  { label: 'All users', value: 'allUsers', role: 'allUsers'}
```

Comment:
all users – is out of the scope, so please remove it

Reply:
done